.travis.yml

```diff
@@ -1,29 +1,42 @@
-# Linux Build Configuration for Travis
-
 language: cpp
 
 os:
   - linux
 
 # Use Ubuntu 14.04 LTS (Trusty) as the Linux testing environment.
-sudo: required
 dist: trusty
+sudo: false
 
-env:
-  # Each line is a set of environment variables set before a build.
-  # Thus each line represents a different build configuration.
-  - BUILD_TYPE=Release
-  - BUILD_TYPE=Debug
+git:
+  depth: false
+
+addons:
+  apt:
+    sources:
+    - llvm-toolchain-trusty
+    - ubuntu-toolchain-r-test
+    packages:
+    - llvm-7-tools
+    - llvm-7-dev
+    - python3
```

Comment:
Do we really need python3 here ?

Reply:
lit is written in python and I doubt the container image has python3 preinstalled... let me check